internal/i18n/en_translations_descriptions.go

```diff
@@ -0,0 +1,544 @@
+	BatchPersistedConfirmed  = ffm("BatchPersisted.confirmed", "The time when the batch was confirmed")
+
+	// Transaction field descriptions
+	TransactionID            = ffm("Transaction.id", "The UUID of the FireFly transaction")
+	TransactionType          = ffm("Transaction.type", "The type of the FireFly transaction")
+	TransactionNamespace     = ffm("Transaction.namespace", "The namespace of the FireFly transaction")
+	TransactionCreated       = ffm("Transaction.created", "The time the transaction was created on this node. Note the transaction is individually created with the same UUID on each participant in the FireFly transaction")
+	TransactionBlockchainIDs = ffm("Transaction.blockchainIds", "The blockchain transaction ID, in the format specific to the blockchain involved in the transaction. Not all FireFly transactions include a blockchain. FireFly transactions are extensible to support multiple blockchain transactions")
+
+	// Operation field description
+	OperationID          = ffm("Operation.id", "The UUID of the operation")
+	OperationNamespace   = ffm("Operation.namespace", "The namespace of the operation")
+	OperationTransaction = ffm("Operation.tx", "The UUID of the FireFly transaction the operation is part of")
+	OperationType        = ffm("Operation.type", "The type of the operation")
+	OperationStatus      = ffm("Operation.status", "The current status of the operation")
+	OperationPlugin      = ffm("Operation.plugin", "The plugin responsible for performing the operation")
+	OperationInput       = ffm("Operation.input", "The input to this operation")
+	OperationOutput      = ffm("Operation.output", "Any output reported back from the plugin for this operation")
+	OperationError       = ffm("Operation.error", "Any error reported back from the plugin for this operation")
+	OperationCreated     = ffm("Operation.created", "The time the operation was created")
+	OperationUpdated     = ffm("Operation.updated", "The last update time of the operation")
+	OperationRetry       = ffm("Operation.retry", "If this operation was initiated as a retry to a previous operation, this field points to the UUID of the operation being retried")
+
+	// BlockchainEvent field descriptions
+	BlockchainEventID         = ffm("BlockchainEvent.id", "The UUID assigned to the event by FireFly")
+	BlockchainEventSource     = ffm("BlockchainEvent.source", "The blockchain plugin or token service that detected the event")
+	BlockchainEventNamespace  = ffm("BlockchainEvent.namespace", "The namespace of the listener that detected this blockchain event")
+	BlockchainEventName       = ffm("BlockchainEvent.name", "The name of the event in the blockchain smart contract")
+	BlockchainEventListener   = ffm("BlockchainEvent.listener", "The UUID of the listener that detected this event, or nil for the built-in BatchPin events in the system namespace")
```

Comment:
It will be nil for all built-in events (ie token events as well, not just BatchPin)